modules/nf-core/pbsv/call/main.nf

```diff
@@ -0,0 +1,50 @@
+process PBSV_CALL {
+    tag "$meta.id"
+    label 'process_single'
```

Comment:
Since this module can use multiple threads consider changing from `process_single` to `process_medium`. I'm not sure how much impact this has since I haven't run pbsv myself. Fig.5 in [this paper](https://www.researchgate.net/publication/343401645_Long-read-based_human_genomic_structural_variation_detection_with_cuteSV) suggests maybe not that much, but they don't report discover and call separately.

Reply:
Okay! Might as well do it, thank you!